codespell_lib/data/dictionary.txt

```diff
@@ -445,6 +445,7 @@ accuratelly->accurately
 accuratley->accurately
 accuratly->accurately
 accuray->accuracy, actuary,
+accured->accrued, accure, occurred,
```

Comment:
Do you really mean `accure`? I couldn't find that either, or do you mean `accrue` in which case I don't think we suggest other variants of the same word normally, or you'd get all the plural bits too.

Reply:
google and my german-to-english dictionary say it is a valid word. I'm gonna double check it with english only dictionaries from now on :-)

Reply:
deleted it.

Reply:
Okay, after a little more research:

My german dictionary and google translate claim, that it means (translated back from german) exact or precise.
On grep.app it finds a lot of accure->occure and some phrases, where I would use get or gather or rephrase them.
f.e. "Failed to accure lock for resource {resource}!"

Reply:
"Failed to accure lock for resource {resource}!" is probably a typo for "Failed to acquire lock for resource {resource}!"

"exact or precise" sounds like the definition for "accurate" so maybe your german-to-english dictionary has a typo! :laughing:

Reply:
Yes acquire was the translation I was missing earlier today :+1: 

Yes, it seems so. Not the first mistake I've found.

Reply:
Added accure->accrue, occur, acquire,

Reply:
FWIW Cambridge and Oxford are my go to British English dictionaries to check.

Reply:
I will use them more often now :-)